graphix/sim/tensornet.py

```diff
@@ -568,7 +572,12 @@ def expectation_value(self, op, qubit_indices, output_node_indices=None, **kwagr
             target_nodes = [output_node_indices[ind] for ind in qubit_indices]
             out_inds = output_node_indices
         op_dim = len(qubit_indices)
-        op = op.reshape([2 for _ in range(2 * op_dim)])
+        op = op.reshape(
+            [
+                2,
+            ]
+            * (2 * op_dim)
+        )
```

Comment:
use `black -l 120`?

Reply:
Yes. If I left this like `op = op.reshape([2,] * (2 * op_dim)`, cannot pass the check for black.